diff --git a/Android.bp b/Android.bp
index 645e2a6..2837c23 100644
--- a/Android.bp
+++ b/Android.bp
@@ -27,10 +27,11 @@ rust_library {
     cargo_pkg_version: "0.3.0",
     srcs: ["src/lib.rs"],
     edition: "2018",
+    no_stdlibs: true,
     rustlibs: [
-        "libbitflags",
-        "liblog_rust",
-        "libzerocopy",
+        "libbitflags-1.3.2",
+        "liblog_rust_nostd",
+        "libzerocopy_nostd",
     ],
     apex_available: [
         "//apex_available:platform",
@@ -48,7 +49,7 @@ rust_test {
     auto_gen_config: true,
     edition: "2018",
     rustlibs: [
-        "libbitflags",
+        "libbitflags-1.3.2",
         "liblog_rust",
         "libzerocopy",
     ],
